website/src/pages/Home/index.tsx

```diff
@@ -272,7 +237,32 @@ const Home: React.FC = () => {
           ><img loading="lazy" src="../Images/svg/left_arrow.svg" alt={t('home.adaptorsTestimonials.previousArrowAlt')} /></div>
         );
     }
-
+    
+    const testimonialSliderSettings = {
+        dots:false,
+        infinite: true,
+        autoplay: true,
+        autoplaySpeed: 4000,
+        speed:500,
+        slidesToShow:1,
+        slidesToScroll: 1,
+        cssEase:"linear",
+        arrows:true,
+        centerMode:true,
+        prevArrow: <SamplePrevArrow />,
+        nextArrow: <SampleNextArrow />,
```

Comment:
Can we please rename `SamplePrevArrow` and `SampleNextArrow`  ==> `PrevArrow` and `NextArrow` corresponding?